### PR TITLE
init commit

### PR DESCRIPTION
**Pull request by Ekamba**

### CODE REVIEW

File: react/week3/src/components/TodoForm.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  };
+
+  return (
+    <form className="form-group">
+      <label>
+        <span>
+          <b>Todo description</b>:
+        </span>
+        <input
+          //className="form-control"
```
Comment (Daryan-Tech) [inline react/week3/src/components/TodoForm.js]: try to always remove the unused or useless line of codes like this comment

**Comment by Daryan-Tech** [comment]
you have done a great job just some small thing that you can do better 👍

File: react/week3/src/App.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  const [todos, setTodos] = useState([]);
+  useEffect(() => {
+    axios
+      .get(
+        "https://gist.githubusercontent.com/benna100/391eee7a119b50bd2c5960ab51622532/raw"
+      )
+      .then(res => {
+        setTodos(res.data);
+      })
+      .catch(err => {});
```
[the remark this answers is not in the document]
Comment (Ekamba) [reply react/week3/src/App.js]: Ok, thanks for advice

**Comment by Ekamba** [comment]
Thanks a lot Daryan-tech for reviewing my homework

**Pull request merged**